folly/test/FixedStringTest.cpp

```diff
@@ -347,6 +347,8 @@ TEST(FixedStringAssignTest, RuntimeAppendString) {
   EXPECT_STREQ("hello world!", a.c_str());
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstring-plus-int"
```

Comment:
Prefer `FOLLY_CLANG_DISABLE_WARNING` from https://github.com/facebook/folly/blob/master/folly/Portability.h#L171

Reply:
Can we instead change the code to avoid this construction? E.g., if this works:

```
static constexpr char s[] = "X world!";
s.append(&s[2u], 5u);
s.append(&s[7u]);
```

Reply:
Yep, that works @yfeldblum 👍